depends/packages/boost.mk

```diff
@@ -31,7 +31,9 @@ $(package)_cxxflags_linux=-fPIC
 $(package)_cxxflags_android=-fPIC
 endef
 
+# Fix unused variable in boost_process, can be removed after upgrading to 1.72
 define $(package)_preprocess_cmds
+	sed -i.old "s/int ret_sig = 0;//" boost/process/detail/posix/wait_group.hpp && \
```

Comment:
Is a tab character placed here intentionally?

Reply:
Just copy-pasted that line... Fixed, since I had to rebase again.